### PR TITLE
update scopes for cloudzero integration

### PR DESCRIPTION
**Pull request by ben-dalton**
### What does this PR do?

Updates the scopes required for the CloudZero integration to include `metrics_read` and `timeseries_query`.

### Motivation

Our customers are looking for more ways to leverage Unit Cost Metrics.  By providing an easy way to pull metrics from an existing Datadog account we can accelerate our end users FINOPS journey. 

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by MatthewNorgren — APPROVED** [review]

**Review by emarsha94 — APPROVED** [review]
Can you confirm that you tested this new scope in your EU and US sandbox?

**Review by bgoldberg122 — APPROVED** [review]

**Comment by ben-dalton** [comment]
@emarsha94 yes, this has been tested

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Can you update the classifier tags in your manifest.json to add: Queried Data Type::Metrics as seen here https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags 

Then we are good to go!

File: cloudzero/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - CloudZero
 
+## 1.1.0 / 2024-07-09
```
Comment (emarsha94) [inline cloudzero/CHANGELOG.md:3]:
```suggestion
## 1.1.0 / 2024-07-17
```

**Comment by ben-dalton** [comment]
Thank you, @emarsha94 . These changes have been made.

**Review by emarsha94 — APPROVED** [review]

**Pull request merged**